datadog_checks_base/datadog_checks/base/utils/metadata/utils.py

```diff
@@ -0,0 +1,8 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+def is_primitive(obj):
+    # https://github.com/python/cpython/blob/4f82a53c5d34df00bf2d563c2417f5e2638d1004/Lib/json/encoder.py#L357-L377
```

Comment:
FWIW you're pointing to the section about keys, but we're handling values, no?

Reply:
Yes true, my goal here was to show what they consider primitive/hashable